game/hiscore.cc

```diff
@@ -9,24 +9,32 @@
 #include <stdexcept>
 
 bool Hiscore::reachedHiscore(unsigned score, SongId songid, unsigned short level, std::string const& track) const {
-	if (score > 10000) throw std::logic_error("Invalid score value");
-	if (score < 2000) return false; // come on, did you even try to sing?
+	if (score > 10000)
+		throw std::logic_error("Invalid score value");
+	if (score < 2000)
+		return false; // come on, did you even try to sing?
 
 	unsigned position = 0;
 	for (auto const& elem: m_hiscore) {
 		if (elem.songid != songid) continue;
 		if (elem.track != track) continue;
 		if (elem.level != level) continue;
-		if (score > elem.score) return true; // seems like you are in top 3!
-		if (++position == 3) return false; // not in top 3 -> leave
+		if (score > elem.score) return true;
+		if (++position == 16) return false;
```

Comment:
magic number, please make a constant for this. For now it looks alright, but once someone starts to refactor it might not be alright anymore